Fix/navigation

PiotrDrechsler opened the pull request.

PiotrDrechsler committed:
Fix/Navigation change for HomeNav

Change name for Navigation

PiotrDrechsler committed:
fix/NavHome

NavHome sounds better :D

PiotrDrechsler committed:
feat/close button to mobile menu

add icon

PiotrDrechsler committed:
feat/close button in navhome + animation

add close button in mobile nav and animate it

PiotrDrechsler committed:
fix/close button in mobile nav

resize close button

PiotrDrechsler committed:
feat/HeaderTablet

create header only for tablet a set media to tablet resolution

PiotrDrechsler committed:
fix/tablet navbar

new hamburger-menu icon
styles for tablet menu
new component for tablet navigation
make homepage nav full RWD

Okazaki92 commented on src/components/NavHome/NavHome.module.css:
```diff
@@ -68,6 +69,22 @@
+    margin-bottom: 70px;
+  }
+
+  .nav .navList button {
+    display: none;
+  }
```
zapomniałeś ten button włączyć w media 1280px i go teraz nie ma

PiotrDrechsler replied:
fixed :D

Okazaki92 commented on src/components/NavTablet/NavTablet.module.css:
```diff
@@ -0,0 +1,33 @@
+    z-index: 999;
+  }
+
+  .navTablet button {
+    font-size: 1rem;
+  }
```
tutaj bym dodał width: 35px i display: flex z align-items:center to wtedy button będzie jak na figmie :D

PiotrDrechsler replied:
Fixed. O teraz super wygląda i nie trzena już poprawiać :D Dzięki!

Okazaki92 reviewed (changes requested):
Kilka małych poprawek opisałem w komentarzach ale mam jeszcze 2 sprawy tutaj:
1. w zakładce Edycje niewidoczny jest dropdown
2. fajnie by było wstawić żeby ten navbar w mobilce i tablecie po kliku na dany odnośnik się chował. Chyba, że za to się dopiero będziesz zabierał 😄 
Reszta wygląda OK 👍

PiotrDrechsler committed:
fix/button display on desktop

PiotrDrechsler committed:
fix/navTablet button

fixed size of button
now is almost 1:1 with sizes on figma

PiotrDrechsler committed:
fix/hamburger-menu and code optimalization

import and install hamburger-react
refactor code for hamburger-react

create new file with li elements of navlink and impor them to NavHome and map them

Okazaki92 reviewed (approved):
Wygląda dobrze :D zatwierdzam. Teraz tylko podłaczyć przekierowania :D
